mlir/include/mlir/Pass/PassManager.h

```diff
@@ -307,7 +307,8 @@ class PassManager : public OpPassManager {
     ///   IR.
     explicit IRPrinterConfig(
         bool printModuleScope = false, bool printAfterOnlyOnChange = false,
-        bool printAfterOnlyOnFailure = false,
+        bool printAfterOnlyOnFailure = false, bool printBeforePass = false,
+        bool printAfterPass = false,
```

Comment:
@joker-eph I try to create C struct of `PassManager::IRPrinterConfig`, but this will change the member variable. I don't know whether it is a good practice.

Reply:
BTW, this PR is only a draft now, don't care about the detail code.